ext/debug/segger/Kconfig

```diff
@@ -25,6 +25,16 @@ config SEGGER_SYSTEMVIEW
 	select THREAD_STACK_INFO
 	select TRACING
 
+config SEGGER_SYSTEMVIEW_START_LOGGING
```

Comment:
"Start Logging" have a large interpretation area. What aboot "BOOT_ENABLE" or something which indicate that the action is on boot time.

Reply:
ok, so I will change it to SEGGER_SYSTEMVIEW_BOOT_ENABLE